npm/test-cli.js

```diff
@@ -25,6 +25,11 @@ module.exports = function (exit) {
     // eslint-disable-next-line no-process-env
     process.env[isWin ? 'userprofile' : 'HOME'] = outDir;
 
+    // Used to indicate the child processes that their execution is during a test
+    // This slightly alters their logic to test some cases
+    // eslint-disable-next-line no-process-env
+    process.env.NEWMAN_TEST_ENV = 'cli';
```

Comment:
This is done here since `NEWMAN_TEST_ENV=cli node test-cli.js` does not work in Windows